<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profil</class>
 <widget class="QWidget" name="Profil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1359</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1361</width>
     <height>741</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(64, 118, 114);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>1231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(122, 31, 52);
font: 600 italic 15pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Profil</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonZurueck">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #3498db;
    color: #ffffff;
    font: 600 italic 15pt &quot;Segoe UI&quot;;
    padding: 10px 20px;
    border: 2px solid #2980b9;
    border-radius: 8px;
}

QPushButton:hover {
    background-color: #2980b9;
}

QPushButton:pressed {
    background-color: #1f618d;
}
</string>
   </property>
   <property name="text">
    <string>Zurück</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameText">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(122, 31, 52);
font: 600 italic 15pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="waehrungText">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>370</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(122, 31, 52);
font: 600 italic 15pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Währung:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="gespartText">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>470</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(122, 31, 52);
font: 600 italic 15pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Gespart:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="gespart">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>470</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(225, 225, 225);
color: rgb(0, 0, 0);
font: 600 italic 15pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>270</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 italic 15pt &quot;Segoe UI&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAenderungSpeicchern">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>580</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {

        background-color: rgb(40, 53, 132);
    color: #ffffff;
    font: 600 italic 15pt &quot;Segoe UI&quot;;
    padding: 10px 20px;
    border: 2px solid #2980b9;
    border-radius: 8px;
}

QPushButton:hover {
    background-color: #2c82c9;
}

QPushButton:pressed {
    background-color: #1869ab;
}
</string>
   </property>
   <property name="text">
    <string>Änderung Speichern</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>370</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
        font: 600 italic 15pt &quot;Segoe UI&quot;;
        color: rgb(0, 0, 0);
    padding: 5px;
    background-color: #f5f5f5;
    selection-background-color: #66bb6a;
    border: 1px solid #707070;
    border-radius: 3px;
}
</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>80</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
    border: 5px solid rgb(85, 85, 127);
    border-radius: 10px;
    text-align: center;
        background-color: rgb(85, 85, 127);
            border: 2px solid white;
    padding: 5px;

}

QProgressBar::chunk {
        background-color: rgb(255, 255, 255);

    width: 10px;
}
</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="level">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>50</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">


QLabel {
        font: 600 italic 25pt &quot;Segoe UI&quot;;
        background-color: rgb(85, 85, 127);
    border-radius: 50%;
    text-align: center;
        color: rgb(255, 255, 255);
    border: 5px solid white;
    padding: 5px;
}
</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>120</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(64, 118, 114);
font: 600 italic 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
